iotdb/src/test/java/org/apache/iotdb/db/engine/bufferwrite/BufferWriteProcessorTest.java

```diff
@@ -194,14 +197,23 @@ public void testWriteAndQuery()
     assertEquals(0, bufferwrite.memoryUsage());
     assertEquals(TsFileIOWriter.magicStringBytes.length, bufferwrite.getFileSize());
     assertEquals(0, bufferwrite.getMetaSize());
+    long lastFlushTime = bufferwrite.getLastFlushTime();
     for (int i = 1; i <= 85; i++) {
       bufferwrite.write(deviceId, measurementId, i, dataType, String.valueOf(i));
       assertEquals(i * 12, bufferwrite.memoryUsage());
     }
+    assertEquals(lastFlushTime, bufferwrite.getLastFlushTime());
     bufferwrite.write(deviceId, measurementId, 86, dataType, String.valueOf(86));
-    assertEquals(true, bufferwrite.isFlush());
+    //assert a flush() is called.
+    assertEquals(false, bufferwrite.getLastFlushTime()==lastFlushTime);
```

Comment:
Here too, if the machine is too powerful, this check will fail.
By the way, using 'assertNotEquals' is more straight-forward.

Reply:
fixed.